mosaic-site/src/components/mdx/code/Code.module.css

```diff
@@ -0,0 +1,12 @@
+.code {
+  --code-padding-vertical: 0.1rem;
+  --code-padding-horizontal: 0.1rem;
+}
+
+.code {
+  font-family: SFMono-Regular, Menlo;
```

Comment:
did we want to use --salt-typography-fontFamily-code ?

Reply:
This is just to match what we currently have on the site which unfortunately comes from Docusaurus and looks slightly different than our `--salt-typography-fontFamily-code` token